cypress/integration/plugins/gantt-chart-dashboards/gantt_ui.spec.js

```diff
@@ -18,6 +18,8 @@ const Y_LABEL = 'A unique label for Y-axis';
 const X_LABEL = 'A unique label for X-axis';
 const DEFAULT_SIZE = 10;
 
+var theVisualizationId = ''
```

Comment:
This creates dependence between the two tests, I may refactor it a bit like this:
```javascript
function createGanttChart(name) {
  cy.get('.euiButton__text').contains('Create ').click({ force: true });
  cy.wait(delay);
  cy.get('[data-test-subj="visTypeTitle"]')
    .contains('Gantt Chart')
    .click({ force: true });
  cy.wait(delay);
  cy.contains(/^jaeger$/).click({ force: true });
  cy.wait(delay);
  cy.contains('Save').click({ force: true });
  cy.wait(delay);
  cy.get('input[data-test-subj="savedObjectTitle"]').type(name);
  cy.wait(delay);
  cy.get('button[data-test-subj="confirmSaveSavedObjectButton"]').click({
    force: true,
  });
  cy.wait(delay);
}

it('Creates and saves a gantt chart', () => {
  createGanttChart("test gantt chart create and save")
  cy.contains('Saved').should('exist');
  cy.visit(`${BASE_PATH}/app/visualize#`);
  cy.contains("test gantt chart create and save")
});

describe('Render and configure a gantt chart', () => {
  const ganttChartName = "test gantt chart render and configure"
  beforeAll(() => {
    cy.visit(`${BASE_PATH}/app/visualize#`);
    createGanttChart(ganttChartName)
  })
  beforeEach(() => {
    cy.visit(`${BASE_PATH}/app/visualize#`);
    cy.contains(ganttChartName).click({ force: true });
  })
})
```
Does it make sense?

Reply:
I think you misunderstood the change.
The point of this change is to enable the direct-navigation to the target chart.
 
 https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/802/files#diff-79b81ed2225f6c543217d6fb6f10db10e4677a47437c0a20c5bb3956c3513e2aR93
 
We admit that this test suite is interdependent; this PR is not intended to correctly unwind this test.  
This PR is only intended to resolve the current brittle-test in the most efficient change possible.

Reply:
Oh, if fixing the interdependent is out of the scope, then I'm fine with it. We may need get back to this later, it looks fragile, I wonder if cypress guarantee the test to be run sequentially.